src/components/smart-container/smart-container.js

```diff
@@ -94,7 +96,12 @@ class SmartContainer extends Component {
     props.clearAccessibleChildren();
     return this.isPortal ? (
       createPortal(
-        <Overlay open onClose={props.onClose} handleKeyDown={this.props.handleKeyDown} addAccessibleChild={this.props.addAccessibleChild}>
+        <Overlay
+          open
+          onClose={props.onClose}
+          handleKeyDown={this.props.handleKeyDown}
+          addAccessibleChild={this.props.addAccessibleChild}
+          label={props.settingsText}>
```

Comment:
add {props.settingsText + " " + 'dialog'}, otherwise the screenreader won’t indicate it is a dialog

Reply:
screen reader already reads "dialog" because of the role="dialog"